assets/src/edit-story/components/library/panes/pageLayouts/pageLayout.js

```diff
@@ -18,32 +18,44 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import { useRef, useState, useCallback } from 'react';
+import { useState, useCallback, useRef } from 'react';
 import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
+import { useFocusOut, useKeyDownEffect } from '../../../../../design-system';
 import { PageSizePropType } from '../../../../types';
 import { PreviewPage, PreviewErrorBoundary } from '../../../previewPage';
-import useFocusOut from '../../../../utils/useFocusOut';
 import { STORY_ANIMATION_STATE } from '../../../../../animation';
 import { KEYBOARD_USER_SELECTOR } from '../../../../utils/keyboardOnlyOutline';
+import useRovingTabIndex from '../../../../utils/useRovingTabIndex';
 import ConfirmPageLayoutDialog from './confirmPageLayoutDialog';
 
 const PageLayoutWrapper = styled.div`
-  position: relative;
+  position: absolute;
+  top: 0;
   height: ${({ pageSize }) => pageSize.containerHeight}px;
   width: ${({ pageSize }) => pageSize.width}px;
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+  transform: ${({ translateX, translateY }) =>
+    `translateX(${translateX}px) translateY(${translateY}px)`};
+
+  ${KEYBOARD_USER_SELECTOR} &:focus {
+    outline: ${({ theme }) => `2px solid ${theme.colors.selection} !important`};
```

Comment:
Do we need the `impotant!` flag here?

Reply:
yeah, there's global settings for outlines that can take priority right now.